.cirrus.yml

```diff
@@ -814,7 +814,7 @@ podman_machine_mac_task:
     clone_script:  # artifacts from osx_alt_build_task
         - mkdir -p $CIRRUS_WORKING_DIR
         - cd $CIRRUS_WORKING_DIR
-        - $ARTCURL/OSX%20Cross/repo/repo.tbz
+        - $ARTCURL/Build%20for%20MacOS%20amd64%2Barm64/repo/repo.tbz
```

Comment:
This will not be correct unless you also backport the task rename but this is overkill so just drop that here

Reply:
I am mean you could have just dropped https://github.com/containers/podman/pull/23240/commits/831baea60e97efe1cb051ccdec02b748786d8bf7 but I well manually reverting works too.